<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightComp</class>
 <widget class="LightGroup" name="LightComp">
  <property name="windowTitle">
   <string>Light</string>
  </property>
  <property name="title">
   <string>Light</string>
  </property>
  <layout class="QFormLayout" name="lightLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="directionLabel">
     <property name="text">
      <string>Direction</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="directionLayout">
     <item>
      <widget class="ValuePicker" name="directionX">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="directionY">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValuePicker" name="directionZ">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="energyLabel">
     <property name="text">
      <string>Energy</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ValuePicker" name="energy">
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.250000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string>Color (RGB)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="colorLayout">
     <item>
      <widget class="QSlider" name="colorR">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="colorG">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="colorB">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValuePicker</class>
   <extends>QDoubleSpinBox</extends>
   <header>RaZor/Interface/ValuePicker.hpp</header>
  </customwidget>
  <customwidget>
   <class>LightGroup</class>
   <extends>QGroupBox</extends>
   <header>RaZor/Interface/Component/LightGroup.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
